tests/tests/cpld_sfs_hydro

```diff
@@ -0,0 +1,89 @@
+#
+#  cpld_control GFSv17 with UGWPv1 test, i.e., GSL orographic drag +
+#  version 1 non-stationary GWD
+#
+
+export TEST_DESCR="Fully coupled FV3-CCPP-MOM6-CICE-CMEPS-WW3 system - C96MX100 - unst WW3"
```

Comment:
I think it would be good to have the description add that this is w/ hydrostatic option.